backend/azureblob/azureblob.go

```diff
@@ -499,6 +526,13 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 			string(azblob.AccessTierHot), string(azblob.AccessTierCool), string(azblob.AccessTierArchive))
 	}
 
+	if opt.PublicAccess == "" {
```

Comment:
If you add the `Default:` in the options above, then you don't need to test for `opt.PublicAccess == ""` as it will always be set.

I'm not sure of the actual value of `azblob.PublicAccessNone` - is it `""`

Reply:
Yes `azblob.PublicAccessNone` does resolve to `''`.